Cargo.toml

```diff
@@ -18,10 +18,13 @@ default = []
 arrayref = "0.3"
 bbs = { version = "0.4.1", default-features = false, features = ["wasm"] }
 console_error_panic_hook = { version = "0.1.1", optional = true }
+hkdf = "0.8"
 js-sys = "0.3"
 rand = { version = "0.7", features = ["wasm-bindgen"] }
+pairing-plus = "0.19"
 serde = { version = "1.0", features = ["derive"] }
 serde-wasm-bindgen = "0.1.3"
+sha2 = "0.8"
```

Comment:
Are we using all of these new dependencies?

Reply:
Not really. BBS crate uses this too.